src/service/print/docxHelper.js

```diff
@@ -270,20 +274,20 @@ async function renderMCQ(question, questionCounter, marks) {
       typeof questionOptions[0][0] === "object"
     ) {
       if (questionOptions[0][0].text) {
-        questionOpt.push(["I.", questionOptions[0][0].text[1]]);
+        questionOpt.push([`${getLanguageKey(question.medium[0].toLowerCase(), 'i')+")"}`, questionOptions[0][0].text[1]]);
         imageProperties.push({
           width: 0,
           height: 0,
         });
       } else {
-        questionOpt.push(["I.", questionOptions[0][0].image]);
+        questionOpt.push([`${getLanguageKey(question.medium[0].toLowerCase(), 'i')+")"}`, questionOptions[0][0].image]);
         imageProperties.push({
           width: questionOptions[0][0].width,
           height: questionOptions[0][0].height,
         });
       }
     } else {
-      questionOpt.push(["I.", questionOptions[0][0]]);
+      questionOpt.push([`${getLanguageKey(question.medium[0].toLowerCase(), 'i')+")"}`, questionOptions[0][0]]);
```

Comment:
question.medium[0]

We should not assume that always Question is mapped to Medium. At present, we have this for Education, but some other adopters may define medium may be not.